src/main/java/teammates/common/util/GaeApi.java

```diff
@@ -0,0 +1,36 @@
+package teammates.common.util;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import com.google.appengine.api.modules.ModulesService;
+import com.google.appengine.api.modules.ModulesServiceFactory;
+
+/**
+ * Calls API functions of Google AppEngine
+ */
+public class GaeApi {
```

Comment:
Actually, `GaeApi` might be too broad a term is the functions only involve versioning. Consider renaming it to something else that captures the versioning functionality?